tracer/samples/RabbitMQ.DistributedTracing/Receive/Receive.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Datadog.Trace" Version="1.19.3" />
+    <PackageReference Include="Datadog.Trace" Version="2.13.0" />
```

Comment:
I assume send.csproj is out of date and needs updating too?

Reply:
Send.csproj doesn't need Datadog.Trace anymore. I'll remove it.